src/frameworks/gatsby.json

```diff
@@ -23,6 +23,11 @@
     "AWS_LAMBDA_JS_RUNTIME": "nodejs14.x",
     "NODE_VERSION": "14"
   },
+  "logo": {
+    "default": "https://framework-info.netlify.app/logos/gatsby/default.svg",
+    "light": "https://framework-info.netlify.app/logos/gatsby/light.svg",
+    "dark": "https://framework-info.netlify.app/logos/gatsby/dark.svg"
```

Comment:
Might not be an issue but do we want to support non-production URLs here? I've used the `DEPLOY_PREVIEW` URL in my own site to test the site prior to live deployment.

Reply:
Addressed in https://github.com/netlify/framework-info/pull/797/commits/20f8e7c410d81229279d4fe45bce6b634ab9070f

Reply:
Nice! Thanks.